### ISSUE TITLE
As a motion-sensitive user, I want my browser reduced motion preference honored and the parallax effect on the site homepage not enabled, so that the parallax doesn't make me feel unwell.

### ISSUE THREAD

**Issue by thatbudakguy** [Princeton-CDH/ppa-django#316]
it looks like we could use the [prefer-reduced-motion](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion) css media feature to remove the parallax effect on the homepage for motion-sensitive users, which is probably a good idea.

**Comment by rlskoeser** [comment]
This should be a simple media query; just don't enable the parallax code if reduced motion is preferred. See js media query here:
https://web.dev/prefers-reduced-motion/

**Comment by rlskoeser** [comment]
Confirmed on the test site using Chrome rendering tools to simulate `prefer-reduced-motion` setting — when set to emulate prefers reduced motion, parallax is not enabled; when no emulation is set, parallax is enabled normally.
